staff/sys/stdin2syslog

```diff
@@ -1,49 +1,22 @@
-#!/usr/bin/env python3
-"""Write stdin logs to OCF's remote syslog server.
+#!/bin/bash
+set -euo pipefail
 
-You can use arbitrary stream names (without configuring them before); each
-stream will show up at syslog:/var/log/remote/<stream>.log.
-"""
-import argparse
-import re
-import subprocess
-import sys
-
-
-VALID_STREAM_NAME = re.compile('^[a-z0-9\-_]+$')
-
-
-class LoggingFailure(Exception):
-    pass
+if [ $# -lt 1 ] || [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
```

Comment:
It just occurred to me: you might want to check `$# -ne 1` instead since the script expects exactly one argument.

Reply:
Good point.